### PR TITLE
refactor(client): get rid of hammerhead.get usage

### PR DESCRIPTION
**Pull request by wentwrong**
## Purpose
As a preparatory step towards migrating from Webmake to Rollup module bundler (#2567), we need to get rid of using the `hammerhead.get` method. It stores a reference to the `require` function and is used for importing internal hammerhead submodules in the client tests. Such functionality cannot be achieved using Rollup, because Rollup doesn't preserve the structure of the exported submodules inside generated bundle.

## Approach
Put all submodules exposed in the tests into the "Hammerhead" class properties. _This class became too large for single file because of many imported submodules. Maybe it's worth to separate into different classes? Any suggestions on how to structure it better would be nice._

### CODE REVIEW

**Comment by wentwrong** [comment]
@testcafe-build-bot retest

**Comment by wentwrong** [comment]
@testcafe-build-bot retest

**Comment by wentwrong** [comment]
@testcafe-build-bot retest

**Comment by wentwrong** [comment]
@testcafe-build-bot retest

File: src/client/index.ts [hunk 1]
```diff
@@ -89,20 +125,25 @@ class Hammerhead {
         };
 
         this.PROCESSING_COMMENTS = {
-            stylesheetStart: StyleProcessor.STYLESHEET_PROCESSING_START_COMMENT,
-            stylesheetEnd:   StyleProcessor.STYLESHEET_PROCESSING_END_COMMENT,
+            stylesheetStart: styleProcessor.STYLESHEET_PROCESSING_START_COMMENT,
```
Comment (miherlosev) [inline src/client/index.ts:128]: Add a an additional namespace:
```js
this.PROCESSING_COMMENTS.stylesheet.start
this.PROCESSING_COMMENTS.stylesheet.end
this.PROCESSING_COMMENTS.script.start
...
```

File: src/client/index.ts [hunk 2]
```diff
@@ -89,20 +125,25 @@ class Hammerhead {
...
+            stylesheetEnd:   styleProcessor.STYLESHEET_PROCESSING_END_COMMENT,
             scriptStart:     SCRIPT_PROCESSING_START_COMMENT,
             scriptEndHeader: SCRIPT_PROCESSING_END_HEADER_COMMENT,
             scriptEnd:       SCRIPT_PROCESSING_END_COMMENT
         };
 
+        this.SCRIPT_PROCESSING_INSTRUCTIONS = SCRIPT_PROCESSING_INSTRUCTIONS;
```
Comment (miherlosev) [inline src/client/index.ts:135]: Restructure in the following way:
```js
this.PROCESSING_INSTRUCTIONS.dom.internal_attributes
this.PROCESSING_INSTRUCTIONS.dom.internal_props
this.PROCESSING_INSTRUCTIONS.dom.script
```

File: src/client/index.ts [hunk 3]
```diff
@@ -129,6 +170,12 @@ class Hammerhead {
             DragDataStore:         this.sandbox.event.DragDataStore
         };
 
+        const processingUtils = {
+            script:       scriptProcessingUtils,
+            header:       headerProcessingUtils,
+            instrumented: instrumentedProcessingUtils
```
Comment (miherlosev) [inline src/client/index.ts:176]: `instrumentation: instrumentationProcessingUtils`

**Review by miherlosev — CHANGES_REQUESTED** [review]
Just a few renames

File: src/client/index.ts (hunk 1 above)
Comment (wentwrong) [reply src/client/index.ts]: Now `PROCESSING_COMMENTS` is not used anywhere in the code, so I just removed it.

**Review by miherlosev — APPROVED** [review]

**Review by LavrovArtem — APPROVED** [review]

**Pull request merged**